src/components/info/UserProfile.js

```diff
@@ -46,10 +47,12 @@ export default function UserProfile() {
     try {
       await usersService.logout();
       await encryptionService.clearStoredAccount();
+      await messagesService.clearLocalMessages();
```

Comment:
we already have all these cleaning inside `userService.logout()`
https://github.com/SAMA-Communications/sama-client/blob/35b68fffd03fa9c2b590aaf9d234fff7fd71cd4c/src/services/usersService.js#L131

so no need to do it here